deploy/crds/mongodb.com_mongodb_crd.yaml

```diff
@@ -49,6 +49,46 @@ spec:
             members:
               description: Members is the number of members in the replica set
               type: integer
+            security:
+              description: Security configures security features, such as TLS, and
+                authentication settings for a deployment
+              properties:
+                tls:
+                  description: TLS configuration for both client-server and server-server
+                    communication
+                  properties:
+                    caConfigMapRef:
+                      description: CaConfigMap is a reference to a ConfigMap containing
+                        the certificate for the CA which signed the server certificates
+                        The certificate is expected to be available under the key
+                        "ca.crt"
+                      properties:
+                        name:
+                          type: string
+                      required:
+                      - name
+                      type: object
+                    certificateKeySecretRef:
+                      description: CertificateKeySecret is a reference to a Secret
+                        containing a private key and certificate to use for TLS The
```

Comment:
[nit] is a period missing between TSL and The?

Reply:
Sure is. Good catch!